### PR TITLE
WIP: Adding members page

### PR DESCRIPTION
**Pull request by shailen-naidoo**
This PR is for the member's page of the FEDSA site

### CODE REVIEW

File: components/Menu.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const html = require('html-template-tag')
+
+const Menu = () => html`
+  <nav class="Menu-list">
+    <div class="Menu-box">
+      <a class="Menu-link" href="/">About</a>
+      <span style="padding: 0 6px;">&nbsp;|&nbsp;</span>
```
Comment (schalkventer) [inline components/Menu.js:7]: Yikes! Can we rather use CSS? Did I put these in? :)

File: pages/members/index.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+          <option>Projects</option>
+          <option>Members</option>
+        </select>
+        <button class="styled">Toggle Dark Mode</button>
+      </div>
+    </div>
+  </header>
+  <main class="Layout-content">
+    <div class="Layout-member">
+      <img  src="https://media-exp1.licdn.com/dms/image/C4D03AQGy-zPHoCnVwQ/profile-displayphoto-shrink_200_200/0?e=1587600000&v=beta&t=iIHNVJ2DeX5Z6_ZDgzDK0ya0NLMl6g_WRgIkBS6C6Es">
```
Comment (schalkventer) [inline pages/members/index.js]: Can you perhaps put these in the repo itself?

File: public/main.css [hunk 3]
```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css?family=Muli&display=swap');
```
Comment (schalkventer) [inline public/main.css:1]: Can you rather import via HTML, more performant.

File: public/main.css [hunk 4]
```diff
@@ -311,65 +321,11 @@ body {
...
-    background: none;
-  }
-  */
-
 button {
     display: inline-block;
     cursor: pointer;
     background: none;
-    font-family: Roboto, Arial, Helvetical, sans-serif;
+    font-family: Muli, Arial, Helvetical, sans-serif;
```
Comment (schalkventer) [inline public/main.css:328]: What is Muli?

File: components/Menu.js (hunk 1 above)
Comment (shailen-naidoo) [reply components/Menu.js:7]: Yeah, It was like that. I will clean up everything I can towards the end. Not to worry :wink:

File: public/main.css (hunk 3 above)
Comment (shailen-naidoo) [reply public/main.css:1]: Yeah will do, was just playing around with a different font :smile:

File: public/main.css (hunk 4 above)
Comment (shailen-naidoo) [reply public/main.css:328]: Just a font I started trying out, I can always go back to Roboto if it does not look that great

**Pull request merged**